### PR TITLE
Added Thanos proposal for sandbox

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by stmcginnis** [comment]
Are we sure there aren't any copyright concerns with this name?

**Comment by bwplotka** [comment]
We performed legal trademark checks and we are not aware of anything that would pose a risk. Slight rename is also an option if required.

**Comment by caniszczyk** [comment]
CNCF will take care of the the legal due diligence so don't worry about it, thanks.

File: proposals/thanos.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+* Fabian Reinartz - Google - Nov 2017 - Jan 2018  
+
+**Current maintainers:**
+
+[Public list](https://github.com/improbable-eng/thanos/blob/master/MAINTAINERS.md)
+
+* Bartek Plotka - Improbable
+* Dominic Green - Improbable
+* Frederic Branczyk - RedHat
+* Giedrius Statkevičius - AdForm
```
Comment (tdabasinskas) [inline proposals/thanos.md]: It should be `Adform`

File: proposals/thanos.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+* 3550+ stars
+* 110+ contributors
+* 400+ forks
+
+**Production usage:**
+
+Companies that are actively using Thanos in production and publicly share that:
+
+* [Alibaba](https://www.alibaba.com/)
+* [AdForm](https://site.adform.com/)
```
Comment (tdabasinskas) [inline proposals/thanos.md]: Same here

File: proposals/thanos.md (hunk 1 above)
Comment (bwplotka) [reply proposals/thanos.md]: done, thanks for pointing out :+1:

**Comment by caniszczyk** [comment]
@monadic @jbeda @lizrice have also agreed to sponsor with @xiang90

**Comment by lukaszgryglicki** [comment]
Is DevStats page immediately needed? I can add it tomorrow. I need a list of GitHub org9s) and/or repo(s) that constitute to this project and artwork merged into `cncf/artwork` repo.

**Comment by caniszczyk** [comment]
@lukaszgryglicki we will work on this over the next week or so, no crash rush, we have to onboard the project first to the sandbox

**Comment by bwplotka** [comment]
All, Thanks for the good feedback on the today's Thanos presentation. There will be recording of that if someone here missed the call.

 As @brancz mentioned during the call we haven't thought about Incubating stage initially as we thought that the Sandbox stage is an entrypoint - so thanks a lot for bringing it up!

By look of it, it seems like we might indeed match all of the requirement of the CNCF Incubating Project as specified here: https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#incubating-stage

This being said, we would be more than happy for you to consider Thanos as Incubating project in CNCF.

**Comment by lukaszgryglicki** [comment]
Thanos DevStats page added: https://thanos.devstats.cncf.io

**Comment by lukaszgryglicki** [comment]
If possible, please ping me when Thanos main repo moves from `improbable-eng/thanos` to `thanos-io` org. I will update DevStats config then. Thanks!

**Comment by bwplotka** [comment]
Thanks @lukaszgryglicki :heart: 

Tracking ticket for movement is here: https://github.com/improbable-eng/thanos/issues/1330

**Comment by lukaszgryglicki** [comment]
Please just mention me once moved. Thanks.

**Comment by bwplotka** [comment]
@lukaszgryglicki Thanos project moved to separate org: https://github.com/thanos-io/thanos

 https://github.com/thanos-io/thanos/issues/1330

**Pull request merged**